### ISSUE TITLE
ForceEqualSignAlignment + SpaceAroundOperators causes infinite loop

### ISSUE THREAD

**Issue by alebruck** [rubocop/rubocop#7422]
## Expected behavior

```
Layout/SpaceAroundOperators:
  Enabled: true
```

It should not report an error if more than a single space is around the operator when `ForceEqualSignAlignment` is in place.

## Actual behavior

The cop forces a single space around the operator even when `ForceEqualSignAlignment` is in place, generating an infinite loop. 

```
Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.
```

## Steps to reproduce the problem

.rubocop.yml
```
Layout/ExtraSpacing:
  ForceEqualSignAlignment: true

Layout/SpaceAroundOperators:
  Enabled: true
```
sample.rb
```
module Foo
  class Bar
    FIRST_GLOBAL  = {
      '.' => :dot
    }.freeze
    SECOND_GLOBAL = 1
  end
end
```

## RuboCop version

```
$ [bundle exec] rubocop -V
0.75.0 (using Parser 2.6.5.0, running on ruby 2.4.1 x86_64-darwin16)
```

ps: I may tackle this issue when expected behavior is confirmed :)

**Comment by noraj** [comment]
Similar problem:

Equals signs are aligned (operator aligned on the right), no complain from rubocop.

```
  s.files         = Dir['bin/*'] + Dir['docs/**/*'] + Dir['lib/**/*.rb']
  s.files        += ['LICENSE.txt', '.yardopts-dev', '.yardopts']
```

Plus sign and equal sign are aligned (operator aligned on the left), rubocop complaining (`Operator `+=` should be surrounded by a single space. (convention:Layout/SpaceAroundOperators)`)

```
  s.files         = Dir['bin/*'] + Dir['docs/**/*'] + Dir['lib/**/*.rb']
  s.files         += ['LICENSE.txt', '.yardopts-dev', '.yardopts']
```

## Config

Which is the default:

```
Layout/ExtraSpacing:
  ForceEqualSignAlignment: false

Layout/SpaceAroundOperators:
  Enabled: true
```

**Comment by noraj** [comment]
unstale
